### DISCUSSION TITLE
1st OWL build, using R pi 5 and the compact design.   Any suggestions?

### DISCUSSION THREAD

**Discussion by damnbuffalo** [geezacoleman/OpenWeedLocator#133] [General]
I've placed most of my orders for the BOM on the compact owl.  I plan to bench test this summer, with plans to expand to 30' rig this winter.  I will be using the pi 5 and have ordered the Owl Pi Hat from JLBPCB.  So far projected costs are around ~350 US Dollars per row w/ one unit per row.  I plan on 3D printing the first set up but would like to use the aluminum extruded housing for the rig.  Can I build the housing out of standard aluminum extruded stock material or do I need to use the SLT to order from a manufacturer?  Looking forward to this build, very excited to employ all of the hard work that as been done!

**Comment by geezacoleman** [comment]
This is great to hear! Thanks for giving the OWL a go for your project. Will you be using it for between row green on brown weed detection? What sort of height/mounting arrangement?

A few things to mention:
1. We found an issue with the OWL Pi Hat on the Pi 5 when under load the voltage drops. It's been fixed on a recent version by @blinken in the latest version and also a more efficient flyback diode for better efficiency. So just make sure you use the most recent iteration.
2. I am in the process of improving the 3D prints in #128 - no drastic changes but just improving the way the rubber water proofing bead sits and the way the camera is supported. I plan on pushing those changes this week.

### Housing
For the housing there are a few different Alu options. If you want a low cost, more DIY approach check out @MisterHW work on an enclosure [from 80 x 80 conduit](https://github.com/MisterHW/Notes/tree/master/RPi_enclosure_80x80). Alternatively, I am considering making a larger order of the official OWL enclosures. Should reduce the unit price by at least half (approx US$40) so can send you some once they arrive? If you're interested, I'll get the backplates made up with whatever connector you decide on (default to the suggested Amphenol). Alternatively, could get some made in the US with that STL or the drawings. Any extrusion you come across is of course fine, would just need to modify the internal tray.

Definitely keep us in the loop with your progress and any issues you run into along the way. Always happy to help and try figure them out for you!
